inbound/request/inbound_handler.py

```diff
@@ -17,11 +21,25 @@ def __init__(self):
         self.edifact_recep_producer = EdifactRecepProducer()
         self.edifact_recep_consumer = EdifactRecepConsumer()
 
-    async def on_message_recieved(self, edifact_message):
-        if edifact_message.message:
+    def message_callback(self, edifact_message):
+        logger.info(f'Message recieved by inbound, message: {edifact_message}')
+        if edifact_message:
             fhir_message = self.edifact_to_fhir.convert_edifact_to_fhir(edifact_message)
             self.inbound_sequence_number_manager.record_sequence_number(fhir_message)
             self.edifact_recep_producer.generate_sequences(fhir_message)
-            self.supplier_incoming_mq.send(fhir_message)
+            loop = asyncio.new_event_loop()
```

Comment:
I'm concerned about this code creating and setting the event loop. set_event_loop is applied globally and will affect other parts of the application